modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/UpdateLog.java

```diff
@@ -67,7 +68,8 @@ interface OnUpdateHandler {
          * An actual handler that will be invoked when new update is appended to the log.
          *
          * @param update A new update.
+         * @param metastoreUpdateTimestamp Timestamp assigned to the update by the metastore.
```

Comment:
I'm pretty sure we always use either `Meta Storage` or `Metastorage` when talking about the Meta Storage, let's not add new terms